triage/models/estimators.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.linear_model import LogisticRegression
+
+from triage.models.transformers import DsappCutOff
+
+class DsappLogisticRegression(BaseEstimator, ClassifierMixin):
```

Comment:
I'd prefer a more descriptive name. Part of the reason for its existence is to build in scaling, so would something like `ScaledLogisticRegression` work?

Also, wouldn't this not work with the Feature Importances PR because it doesn't inherit from LogisticRegression? Is this an argument for other logic in that PR, like inspecting the coef_ and intercept_? Maybe we can implement `feature_importances_` here using that method?